cardano-config/cardano-config.cabal

```
@@ -0,0 +1,82 @@
+cabal-version: 3.0
+name: cardano-config
+version: 0.1.0.0
+synopsis: Cardano-node configuration parsing
+description:
+  Datatypes and parsers for optparse-applicative and from YAML/JSON for the @cardano-node@ configuration.
+
+license: Apache-2.0
+license-file: LICENSE
+author: Javier Sagredo
+maintainer: javier.sagredo@iohk.io
+copyright:
+  INTERSECT 2025
+
+category: Data
+build-type: Simple
+extra-doc-files: CHANGELOG.md
+
+common warnings
+  ghc-options:
+    -Wall
+    -Wunused-packages
+
+library
+  import: warnings
+  exposed-modules:
+    Cardano.Node.Configuration
```

Maybe
```suggestion
    Cardano.Configuration
```
?

`Cardano.Node...` would suggest that the modules are placed in `cardano-node` package.